app/util/exceptions.py

```diff
@@ -17,11 +18,20 @@ def exception_handler(exc, context):
             status=status.HTTP_409_CONFLICT,
         )
 
+    if not settings.DEBUG and not response:
+        message = (
+            str(exc)
```

Comment:
Vil ikke denne føre til noen lite brukervennlige meldinger? Feilen vil logges uansett så jeg tenker det er greit å kun ha en default melding. Hva tenker du?

Reply:
Er litt usikker kan se begge sidene, for lurer på om vi får bare mld når vi parser det til string som det der. Hva tenker du @olros skal vi alltid forskire oss om at vi får en mld hvis det skjer en uventet feil

Reply:
Tja, usikker. I og med at vi får feilen til Sentry så heller jeg mot at vi nok bare kan vise en standard melding a la "Noe gikk riv ruskende galt i våre interne systemer"

Reply:
Da går vi for det